_episodes/01-plone.md

```diff
@@ -83,39 +85,39 @@ Editing a file:
 
 When in doubt: please ask OTNDC staff to help you find the newest file or export a clean file from the database for you, so you do not accidently use an outdated metadata sheet for mission planning. This will make everyone's lives easier!
 
 ### Detection Extracts
 
-OTN and its partner Nodes create Detection Extracts on a semi-regular basis (approximately every 4 months) following a cross-Node coordinated detection matching event known as a Data Push. These Detection Extract files contain only the detections for the year corresponding to the suffix on the file name. See the detailed [detection extract documentation](https://members.oceantrack.org/data/otn-detection-extract-documentation-matched-to-animals) for more information.
+OTN and its partner Nodes create Detection Extracts on a regular basis, every 4 months (Push months are February, June, and October) following a cross-Node coordinated detection matching event known as a Data Push. These Detection Extract files contain only the detections for the year corresponding to the suffix on the file name. See the detailed [detection extract documentation](https://members.oceantrack.org/data/otn-detection-extract-documentation-matched-to-animals) for more information.
 
-If you are ever interested in learning more about what a project has detected, you are welcome to check out the content in the Detection Extracts. Reminder that you **can not** share this information without reviewing the [OTN Data Policy](https://members.oceantrack.org/data/policies) or contacting OTNDC staff for guidance.
+If you are ever interested in learning more about what a project has detected, you are welcome to check out the content in the Detection Extracts. Reminder that you **can not share this information** without reviewing the [OTN Data Policy](https://members.oceantrack.org/data/policies) or contacting OTNDC staff for guidance.
 
 ### Equipment
 
-This subfolder is the location where Shipping Lists can be posted and reviewed, if relevant. The shipping/inventory tracking workflow has moved to the [SnipeIT inventory system](https://ops.oceantrack.org/snipeit/).
+This subfolder is the location where Shipping Lists can be posted and reviewed, if relevant. The shipping/inventory tracking workflow has moved to the [SnipeIT inventory system](https://ops.oceantrack.org/snipeit/) and [Asana](https://app.asana.com/0/1203822208185209/1203822208185209).
 
 ### Background
 
-This subfolder is the location where OTN Project Management staff will upload loan agreements, project plans and legal paperwork. This folder is only visible to OTN staff members by default.
+This subfolder is the location where OTN Project Management staff will upload loan agreements, project plans and legal paperwork if relevant. This folder is only visible to OTN staff members by default.
```

Comment:
above you have used the oxford comma, so there should be a comma after 'project plans'

Reply:
I think there should also be a comma before 'if relevant'